plezanje-net/api
# Add extra info about image sizes and aspect ratios to image entity.

Also define storage path in env.

Frontend needs info about max possible image width and about image aspect ratio, so these two fields are being added to image entity and read from the actual image 'originals'.

To test this:
- npm install to get sharp (image editing library)
- download all images from the server (fullsizes) and store them locally
- set storage_path variable in .env
- run typeorm migration

Observe that all images now have above two columns set.
